client/scripts/views/pages/new_proposal/new_proposal_form.ts

```diff
@@ -365,28 +365,31 @@ const NewProposalForm = {
           const aaveProposal = vnode.state.aaveProposalState[i];
           if (aaveProposal.target) {
             targets.push(aaveProposal.target);
+          } else {
+            throw new Error(`No target for Call ${i + 1}`);
           }
+
           if (aaveProposal.value) {
             values.push(aaveProposal.value);
+          } else {
+            throw new Error(`No value for Call ${i + 1}`);
           }
+
           if (aaveProposal.calldata) {
             calldatas.push(aaveProposal.calldata);
+          } else {
+            throw new Error(`No calldata for Call ${i + 1}`);
           }
-          if (aaveProposal.signature) {
-            signatures.push(aaveProposal.signature);
-          }
+
           if (aaveProposal.withDelegateCall) {
             withDelegateCalls.push(aaveProposal.withDelegateCall);
+          } else {
+            throw new Error(`No delegate call selected for Call ${i + 1}`);
```

Comment:
withDelegateCalls should default to false if not provided (it should always be provided, is my sense, given the UI).